enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -374,6 +400,264 @@ func (i *InvocationStatService) GetInvocationStatBaseQuery(aggColumn string) str
 	return q
 }
 
+func getTableForMetric(metric *sfpb.Metric) string {
+	switch metric.MetricType.(type) {
+	case *sfpb.Metric_Execution:
+		return "Executions"
+	default:
+		return "Invocations"
+	}
+}
+
+func (i *InvocationStatService) getMetricRange(ctx context.Context, table string, metric string, whereClauseStr string, whereClauseArgs []interface{}) (int64, int64, int64, error) {
```

Comment:
nit: I feel if we define a go struct and return the struct here it would be more readable?

Reply:
Sure, done